### DISCUSSION TITLE
[Feature Request] Add Search Bar To Official site

### DISCUSSION THREAD

**Discussion by ordinarylq** [youzan/vant#11778] [General]
### What problem does this feature solve?

[Official Site](https://vant-contrib.gitee.io/vant/)
Developers can search components and api by using search bar.

### What does the proposed API look like?

A search bar at the nav bar would be fine.

**Comment by chenjiahan** [comment]
We used to use [docsearch](https://docsearch.algolia.com/) to provide search capabilities, but its crawler doesn't work after the site was refactored by Vite, and it's hard to debug. So we removed the search bar from the document site.

I will move this issue to the discussion and look forward to everyone's suggestions for a suitable search.
